spec/enumerables_spec.rb

```diff
@@ -0,0 +1,219 @@
+    end
+
+    let(:expected_array) { array.my_none { |value| value < 0 } }
+
+    it 'Returns true if none of the elements fill criteria' do
+      expect(expected_array).to eql(true)
+    end
+
+    it 'Returns true if none of the elements fill criteria (negative)' do
+      expect(expected_array).not_to eql(false)
+    end
+  end
+
+  describe '#my_count' do
+    it 'Returns the size of the array if no block is given' do
+      expect(array.my_count).to eql(7)
+    end
+
+    it 'Returns the size of the array if no block is given (negative)' do
+      expect(array.my_count).not_to eql(0)
+    end
+
+    it 'Returns how many elements comply with the block' do
+      expect(array.my_count(&:even?)).to eql(4)
+    end
+
+    it 'Returns how many elements comply with the block (negative)' do
+      expect(array.my_count(&:even?)).not_to eql(0)
+    end
+
+    it 'Returns the number of elements equal to the argument if an argument is given' do
+      expect(array.my_count(5)).to eql(1)
+    end
+
+    it 'Returns the number of elements equal to the argument if an argument is given (negative)' do
+      expect(array.my_count(5)).not_to eql(5)
+    end
+  end
+
+  describe '#my_map' do
+    it 'Returns enumerator if no block given' do
+      expect(array.my_map).to be_a(Enumerator)
+    end
+
+    it 'Returns enumerator if no block given (negative)' do
+      expect(array.my_map).not_to be_a(String)
+    end
+
+    it 'Returns a modified array if block is given with instructions' do
+      expect(array.my_map { |value| value * 2 }).to eql([2, 4, 10, 16, 14, 196, 8])
+    end
+
+    it 'Returns a modified array if block is given with instructions (negative)' do
+      expect(array.my_map { |value| value * 2 }).not_to eql(array)
+    end
+  end
+
+  describe '#my_inject' do
+    it 'Gets an initial number and a symbol and operates the array' do
+      expect(array.inject(5, :+)).to eql(130)
+    end
+
+    it 'Gets an initial number and a symbol and operates the array (negative)' do
+      expect(array.inject(5, :+)).not_to eql(10)
+    end
+
+    it 'Gets a symbol and uses it to operate' do
+      expect(array.inject(:+)).to eql(125)
+    end
+
+    it 'Gets a symbol and uses it to operate (negative)' do
+      expect(array.inject(:+)).not_to eql(200)
+    end
+
+    it 'Gets an initial number and operates with the block instructions' do
+      expect(array.inject(5) { |acumulator, value| acumulator + value }).to eql(130)
+    end
+
+    it 'Gets an initial number and operates with the block instructions (negative)' do
+      expect(array.inject(5) { |acumulator, value| acumulator + value }).not_to eql(150)
+    end
+
+    it 'Gets a symbol and uses it to operate' do
+      expect(array.inject { |accumulator, value| accumulator + value }).to eql(125)
+    end
+
+    it 'Gets a symbol and uses it to operate (negative)' do
+      expect(array.inject { |accumulator, value| accumulator + value }).not_to eql(200)
+    end
+  end
```

Comment:
You were suppose to test my_inject. But I can see inject. So replace inject with my_inject.